guides/source/5_0_release_notes.md

```diff
@@ -691,6 +708,9 @@ Please refer to the [Changelog][active-model] for detailed changes.
     [activemodel-serializers-xml](https://github.com/rails/activemodel-serializers-xml) gem.
     ([Pull Request](https://github.com/rails/rails/pull/21161))
 
+*   Removed `ActionController::ModelNaming` module.
+    ([Pull Request](https://github.com/rails/rails/pull/18194))
```

Comment:
Don't think it should be part of release notes, its din't have a deprecation or changelog entry